dev/AppLifecycle/AppInstance.cpp

```diff
@@ -308,51 +316,67 @@ namespace winrt::Microsoft::Windows::AppLifecycle::implementation
     {
```

Comment:
Note that `GetActivatedEventArgs` can throw if the command line is malformed. Since the command line is untrusted, shouldn't this problem be dealt with in a non-exceptional way?